mr_adc_compute.py

```diff
@@ -62,6 +59,13 @@ def kernel(mr_adc):
     # Compute amplitudes
     mr_adc_amplitudes.compute_amplitudes(mr_adc)
 
+    # Compute CVS integrals and amplitudes
+    if mr_adc.method_type == "cvs-ip":
+        mr_adc_integrals.compute_cvs_integrals_2e_incore(mr_adc)
+        mr_adc_amplitudes.compute_cvs_amplitudes(mr_adc)
+
+        del(mr_adc.rdm.ccccaaaa)
```

Comment:
We should find a better place for deleting 4-RDM. May be put it at the end of amplitudes function until we get efficient 4-RDM implemented?

Reply:
I absolutely agree. Moved to the end of the calculation of the amplitudes.